$MirOS: ports/devel/subversion/patches/patch-apr_build_apr_rules_mk_in,v 1.1.7.1 2005/03/18 15:45:35 tg Exp $
--- build/apr_rules.mk.in.orig	2006-03-07 16:51:46.000000000 +0100
+++ build/apr_rules.mk.in	2006-10-03 17:31:48.000000000 +0200
@@ -27,6 +27,8 @@
 apr_builddir=@apr_builddir@
 apr_builders=@apr_builders@
 
+top_builddir=$(apr_builddir)
+
 # Some layouts require knowing what version we are at.
 APR_MAJOR_VERSION=@APR_MAJOR_VERSION@
 APR_DOTTED_VERSION=@APR_DOTTED_VERSION@
